routines/SAMINOT1.m

```diff
@@ -388,20 +388,31 @@ d OUT("")
  d OUT(" ")
  d OUT("Shared Decision Making: ")
  d OUT(" ")
- d OUT("Veteran of age and exposure to cigarette smoke as described above, and without")
- d OUT("a current diagnosis or obvious symptoms suggestive of lung cancer, has been")
- d OUT("educated today about the estimated risk for lung cancer, the possibility of")
- d OUT("cure or life prolonging if an early lung cancer were to be found during")
- d OUT("screening, the possibility of imaging abnormalities not being lung cancer, the")
- d OUT("possibility of complications from additional diagnostic procedures, and the")
- d OUT("approximate amount of radiation exposure associated with each screening")
- d OUT("procedure.  In addition, the Veteran has been educated today about the")
- d OUT("importance of adhering to annual lung screening, the possible impact of other")
- d OUT("medical conditions on the overall health status, the importance of avoiding")
- d OUT("exposure to cigarette smoke, available tobacco cessation programs and")
- d OUT("available lung screening services at the Phoenix VA.  Education material was")
- d OUT("provided to the veteran.  Based on this information, the Veteran has opted")
- d OUT("for: ")
+ d OUT("Veteran of age and exposure to cigarette smoke as described above, and without a ")
+ d OUT("current diagnosis or obvious symptoms suggestive of lung cancer, has been educated ")
+ d OUT("today about the estimated risk for lung cancer, the possibility of a cure or life ")
+ d OUT("prolonging if an early lung cancer were to be found during screening, the possibility of ")
+ d OUT("imaging abnormalities not being lung cancer, the possibility of complications from ")
+ d OUT("additional diagnostic procedures, and the approximate amount of radiation exposure ")
+ d OUT("associated with each screening procedure. In addition, the Veteran has been educated ")
+ d OUT("today about the importance of avoiding exposure to cigarette smoke, available tobacco ")
+ d OUT("cessation programs and available lung screening services at this VA facility. Education ")
+ d OUT("material was provided to the veteran.")
+ ;d OUT(" ")
```

Comment:
Why not just delete all these comments? Git maintains the history of the changes so you don't have to keep this cluttering the code base.

Reply:
I predict that Phoenix will want their own language in their CPRS notes, to be consistent with what they've been doing for years. So I left their language in the code, ready to be conditionally delivered.

Reply:
One possible solution that might be cleaner, is to make all of the text a double colon comment (;; ...), having the different sets of text that might be wanted, stored under separate entry tags, and use $TEXT to grab and write out the correct text, with a flag in a configuration global allowing sites to use whatever text they want of the options.